index.html

```diff
@@ -91,70 +108,70 @@ <h3>Step 3.</h3>
 
   &lt;/body&gt;
   &lt;/html&gt;
-</pre>
-
-    <h3>Step 4.</h3>
-    <p>
-      Much like the previous step, create a <span class="mono">.css</span> file with your name. You can then link it in
-      your <span class="mono">.html</span> file inside the <span class="mono">&lt;head&gt;</span> element with something
-      like:
-    </p>
-
-    <code>
+</pre
+      >
```

Comment:
what crazy linter you got putting closing tags on a new line? 😂

Reply:
good question